File: Backend/src/main/java/com/unida/multitoolshop/model/Order.java
package com.unida.multitoolshop.model;

import com.unida.multitoolshop.enumerations.OrderStatus;
import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
public class Order {
    private int id;
    private MultiToolSet multiToolSet;
    private Customer customer;
    private List<SetOption> options;
    private Float totalPrice;
    private OrderStatus status;
    private Date created;
}

File: Frontend/src/app/models/order.ts
import { Customer } from "./customer";
import { ProductSet } from "./product-set";
import { SetOption } from "./set-option";

export interface Order {
    id?:number,
    customer:Customer,
    multiToolSet:ProductSet|null,
    options:SetOption[],
    totalPrice:number,
    status?:string,
    created?:Date
}
